pull_request_template.md

```diff
@@ -0,0 +1,41 @@
+## [{ticket number/type}:] {title}
+
+### Description :memo:
+{Description}
+
+**Type of change**
+
+*Delete options that are not relevant.*
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
```

Comment:
I never really understood the 'Breaking change ' option 😅 why are we introducing code that would break the provider 🤔 Or am I reading this incorrectly

Reply:
The way I use "Breaking change" is in the sense of an API contract.

Say for instance - something in the public layer of functions a user would use changes and it breaks backwards compatibility.

Made up example:

Version 1 of SDK: .init(string apiKey) requires the API key 
Version 2 of SDK: .init(string apiKey) is removed and there's no only a .init() and the API is configured elsewhere.

If someone using V1 would just update the dependency to RG4N to V2 in their own application, this change would be breaking their application because we've "broken" the API contract.

Reply:
Ahhh, that makes a lot of sense now. Thank you!